docs/remove-ads.md

```diff
@@ -112,15 +115,11 @@ window.addEventListener('load', () => {
 })
 ```
 
-As soon as the page loads, any visitor who does not have Web Monetization
-(`!document.monetization`) sees ads immediately.
-
-If the visitor _does_ have Web Moentization we start our three second timer.
-When the timer is up we check whether the `monetizationstart` event has fired.
-If Web Monetization hasn't initialized, we show them the ads.
+If the visitor _does_ have Web Monetization, the three-second timer starts.
+We check whether the `monetizationstart` event has fired when the timer is up.
+If Web Monetization hasn't initialized, the visitor is shown the ads.
 
 > You might think of using `document.monetization.state` instead of remembering
-> `hasPaid`. But the state can go back to being stopped or pending if the user
-> backgrounds the tab. If they background your tab when the 3 seconds are over
-> then a legitimately Web Monetizated user might would get shown ads! So
-> keeping a `hasPaid` variable is better.
+> `hasPaid`. But, the state can go back to being `stopped` or `pending` if the user
```

Comment:
Given this, should this be tracked on the document.monetization object ?

Reply:
I would say similar to "total," if it can be derived easily we don't need to include it in the WM API. This also gives more control to the page on what the scope of "hasPaid" is.